client/src/App.css

```diff
@@ -107,7 +98,11 @@
 }
 
 .body {
-  margin-top: 150px;
+  font-family: "Roboto", sans-serif !important;
```

Comment:
i'll have same thing here but it's better without !important like this  " font-family: 'Roboto', sans-serif;" aome thing still need to be another font and particularely - code illustration will have some font like VS code i hope later

Reply:
There is a 'Unclosed block
1:43:41 AM:   143 |   background-color: #f5f6fa;
1:43:41 AM:   144 | }
1:43:41 AM: > 145 | .cyf-log {
1:43:41 AM:       | ^
1:43:41 AM:   146 |   width: 250px;
1:43:41 AM:   147 |   height: 112px;' i fixed it in my pull next to this one i think you can merge both just doing nothing and it will work